lib/expansion_tile_card.dart

```diff
@@ -144,6 +145,11 @@ class ExpansionTileCard extends StatefulWidget {
   /// If null, defaults to Theme.of(context).cardColor.
   final Color expandedColor;
 
+  ///The color of the text of the expended card
+  ///
+  ///If null, defaults to Theme.of(context).accentColor.
```

Comment:
```suggestion
  /// The color of the text of the expended card
  ///
  /// If null, defaults to Theme.of(context).accentColor.
```
Nit: Space before comments.

Reply:
Looks good. Thanks